src/celengine/category.cpp

```diff
@@ -5,9 +5,10 @@
 #include <initializer_list>
 #include <utility>
 
+#include <celutil/associativearray.h>
 #include <celutil/gettext.h>
-#include "hash.h"
-#include "value.h"
+
+namespace util = celestia::util;
```

Comment:
as above

Reply:
Not so useful in this file because of the small number of individual usages, have updated some of the other ones.